app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -177,6 +204,47 @@ class Scene extends React.Component {
     this.timer = setTimeout(this.addBlocks, 200);
   };
 
+  addToOverlay(body) {
+    if (!body || !body.render.sprite.texture || body.onOverlay) {
+      return;
+    }
+    const ctx = this.overlay.current.getContext('2d');
+    const { texture } = body.render.sprite;
+    const { sprite } = body.render;
+    ctx.translate(body.position.x, body.position.y);
+    ctx.rotate(body.angle);
+
+    const img = this.getTexture(texture);
+    ctx.drawImage(
+      img,
+      img.width * -sprite.xOffset,
+      img.height * -sprite.yOffset,
+    );
+
+    ctx.rotate(-body.angle);
+    ctx.translate(-body.position.x, -body.position.y);
+    const b = body;
+    b.onOverlay = true;
+  }
+
+  prepareOverlay() {
+    const allBodies = Composite.allBodies(this.engine.world);
+    for (let i = 0; i < allBodies.length; i += 1) {
+      if (allBodies[i].isStatic) {
+        this.invisify(allBodies[i]);
+        this.addToOverlay(allBodies[i]);
+      }
+    }
+  }
+
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
what is this for?

Reply:
I wanted `invisify()` to be class member function but it doesn't need to be because it never uses `this`. So if you think it shouldn't be a member function I can change it.

Reply:
You can just move it to the class. There isn't any value for a funciton that does nothing with the class to be namespaced within the class.